stock_inventory_lockdown/models/stock_location.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
+
+
+class StockLocation(models.Model):
+    """Refuse changes during exhaustive Inventories"""
+    _inherit = 'stock.location'
+    _order = 'name'
+
+    @api.multi
+    @api.constrains('location_id')
+    def _check_inventory_location_id(self):
+        """Error if an inventory is being conducted here"""
+        location_inventory_open_ids = self.env['stock.inventory'].sudo().\
+            _get_locations_open_inventories(self.mapped('location_id').ids)
+        if location_inventory_open_ids:
+            raise ValidationError(
+                _('An inventory is being conducted at this location'))
+
+    @api.multi
+    def unlink(self):
+        """Refuse unlink if an inventory is being conducted"""
+        location_inventory_open_ids = self.env['stock.inventory'].sudo(). \
```

Comment:
This is not working as expected, maybe due to the space after `sudo()`, please remove it.